lib/admin/components/UserList.js

```diff
@@ -132,6 +151,21 @@ class UserList extends Component<Props, State> {
               </span>
               : <span style={headerStyle}>(No results to show)</span>
             }
+            <div className='pull-right'>
+              {this.messages('perPage')}{' '}
+              <DropdownButton
+                id='per-page-dropdown'
+                title={perPage}
+                onSelect={this._setUserPerPage}>
+                {// Render users per page options.
+                  [10, 25, 50, 100].map(val =>
+                    <MenuItem key={`option-${val}`} eventKey={val}>
+                      {val} {val === perPage ? <Icon type='check' /> : null}
```

Comment:
Inconsistent with the feed version selector, where the check mark appears to the left of the selection.

Reply:
Yes consistent with the feed source 'sort by' dropdown in the project view... what to do...

Reply:
Lemme know what you think of bc0eff59ba7387aa53eb080f7d792fcae6093fa8